napari/components/axes.py

```diff
@@ -32,81 +31,11 @@ class Axes:
         If axes have arrowheads or not.
```

Comment:
Maybe swap order of `arrows` and `background_color` to match the order of parameter input?

Reply:
done. we could make alphabetical at some point if we wanted. not sure if parameter input order matters. thoughts @jni?